workflow/rocoto/workflow_xml.py

```diff
@@ -156,11 +156,21 @@ def _write_crontab(self, crontab_file: str = None, cronint: int = 5) -> None:
             replyto = ''
 
         strings = ['',
-                   f'#################### {pslot} ####################',
-                   f'MAILTO="{replyto}"',
-                   f'{cronintstr} {rocotorunstr}',
-                   '#################################################################',
-                   '']
+            f'#################### {pslot} ####################',
+            f'MAILTO="{replyto}"'
+            ]
+       #AWS need 'SHELL', and 'BASH_ENV' defined, or, the crontab job won't start.
+        if os.environ.get('PW_CSP', None) in ['aws', 'azure', 'google']:
+            strings.extend(
+                [
+                f'SHELL="/bin/bash"',
+                f'BASH_ENV="/etc/bashrc"'
+                ])
+        strings.extend(
+            [
+            f'{cronintstr} {rocotorunstr}',
+            '#################################################################',
+            ''])
```

Comment:
Fix `pycodestyle` issues
```suggestion
                   f'#################### {pslot} ####################',
                   f'MAILTO="{replyto}"'
                   ]
        # AWS need 'SHELL', and 'BASH_ENV' defined, or, the crontab job won't start.
        if os.environ.get('PW_CSP', None) in ['aws', 'azure', 'google']:
            strings.extend([
                            f'SHELL="/bin/bash"',
                            f'BASH_ENV="/etc/bashrc"'
                           ])
        strings.extend([
                        f'{cronintstr} {rocotorunstr}',
                        '#################################################################',
                        ''])
```

Reply:
The pynorms errors fixed.